src/argus_htmx/incidents/views.py

```diff
@@ -31,7 +30,6 @@ def prefetch_incident_daughters():
             "incident_tag_relations__tag",
             "events",
             "events__ack",
-            "events__actor",
```

Comment:
It will actually be necessary to prefetch `events__actor__group` iff showing acks by group.